catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/util/impl/SourcePoller.java

```diff
@@ -97,48 +94,14 @@
 
   private volatile List<CatalogStore> catalogStores = Collections.emptyList();
 
-  private final ListeningExecutorService availabilityChecksThreadPool;
-
-  private final ScheduledExecutorService pollAllSourcesThreadPool;
-
-  private ScheduledFuture pollAllSourcesFuture;
-
-  private final Cache<SourceKey, SourceStatus> cache;
-
-  public SourcePoller(
-      ExecutorService availabilityChecksThreadPool,
-      ScheduledExecutorService pollAllSourcesThreadPool) {
-    this(
-        availabilityChecksThreadPool,
-        pollAllSourcesThreadPool,
-        DEFAULT_POLL_INTERVAL,
-        DEFAULT_POLL_INTERVAL_TIME_UNIT,
-        DEFAULT_TIMEOUT,
-        DEFAULT_TIMEOUT_TIME_UNIT);
-  }
-
   /** Starts a process to periodically call {@link #pollAllSources()} */
-  @VisibleForTesting
-  SourcePoller(
-      ExecutorService availabilityChecksThreadPool,
-      ScheduledExecutorService pollAllSourcesThreadPool,
-      final long pollInterval,
-      final TimeUnit pollIntervalTimeUnit,
-      final long timeout,
-      final TimeUnit timeoutTimeUnit) {
+  public SourcePoller(
+      final ExecutorService availabilityChecksThreadPool,
+      final ScheduledExecutorService pollAllSourcesThreadPool) {
     this.availabilityChecksThreadPool =
         MoreExecutors.listeningDecorator(availabilityChecksThreadPool);
     this.pollAllSourcesThreadPool = pollAllSourcesThreadPool;
 
-    LOGGER.debug(
-        "Scheduling ExecutorService at fixed rate of {} {} with an initial delay of {}",
-        pollInterval,
-        pollIntervalTimeUnit,
-        pollInterval);
-    pollAllSourcesFuture =
-        pollAllSourcesThreadPool.scheduleAtFixedRate(
-            this::pollAllSources, pollInterval, pollInterval, pollIntervalTimeUnit);
```

Comment:
❓ Don't we still want to schedule the polling here and keep the log. I think the log should be trace level, however.